### PR TITLE
Feature/document mesh tenant

### PR DESCRIPTION
**Pull request by JohannesRudolph**

### CODE REVIEW

**Comment by JohannesRudolph** [comment]
Note: PR build failed because... Azure 

```
/home/travis/build/meshcloud/meshcloud-docs/docs/meshstack.azure.index.md

  172:131  error  https://developer.microsoft.com/en-us/graph/graph-explorer is dead. (500 Internal Server Error)  no-dead-link

  175:10   error  https://developer.microsoft.com/en-us/graph/graph-explorer is dead. (500 Internal Server Error)  no-dead-link

```

File: docs/meshcloud.customer.md [hunk 1]
```diff
@@ -3,29 +3,27 @@ id: meshcloud.customer
...
-
-A meshCustomer is automatically created for you when you [sign up](meshcloud.profile.md) for a new account. A meshCustomer usually represents your company or department. Within a meshCustomer you manage multiple [projects](meshcloud.project.md), your users and their access rights and maintain general data like addresses and payment methods.
+A meshCustomer usually represents a product team or department. Within a meshCustomer you manage multiple [meshProjects](meshcloud.project.md), your users and their access rights and maintain general data like addresses and payment methods.
 
 ## Customer Settings
 
 General information of a meshCustomer (like its name) can be edited here. The customer identifier is also shown here, but it can never be changed, as it is used as an immutable identifier of the meshCustomer for its representation in the different cloud platforms.
 
-If applicable in your meshcloud installation, also a Tax ID can be defined here for your meshCustomer.
+Depending on the requirements of your organization, you may be able to edit additional customer metadata.
```
Comment (chewbeccy) [inline docs/meshcloud.customer.md]: You can customize the customer according to your needs by setting and editing additional customer metadata.

File: docs/meshcloud.index.md [hunk 2]
```diff
@@ -10,11 +10,11 @@ Here, you can find information on how to use meshcloud and an explanation of the
 
 Quite a bit of complexity in multi-cloud management and multi-cloud governance comes from the use of different and inconsistent terminology between cloud platforms. meshcloud rectifies this using a unified language describing organization and governance structures independent of individual cloud platform implementations. We call this language the **mesh model**. To resolve ambiguity we're prefixing entities in the mesh model using the `mesh` prefix.
 
-The figure below explains the relation of the most important concepts in the mesh model. The table has a summary
+The figure and table below explain the relation of the most important concepts in the mesh model.
```
Comment (chewbeccy) [inline docs/meshcloud.index.md]: meshModel

File: docs/meshcloud.customer.md [hunk 3]
```diff
@@ -17,12 +17,49 @@ If you would like to give others access to your meshCustomer and the related mes
 
 ## Assign User Roles
 
-You can change the group in the dropdown and save the changes via the *disc* icon. The following groups are available:
+You can change the meshCustomer role for each user in the dropdown and save the changes via the *disc* icon. The following roles are available:
```
Comment (chewbeccy) [inline docs/meshcloud.customer.md]: You can change the role of each user associated with a meshCustomer in the dropdown and save the changes via  *disc* icon. The following roles are available:

**Pull request merged**